Sources/gg.datagram.web-requests.sdPlugin/app.js

```diff
@@ -13,13 +13,45 @@ function connected(jsn) {
  *       url?: string,
  *       method?: string,
  *       body?: string|null,
+ *       contentType?: string|null,
+ *       headers?: string|null,
+ *       body?: string|null,
  *     }
  *   },
  * }} data
  */
 function sendHttp(data) {
-    const { url, method, body } = data.payload.settings;
-    log('sendHttp', { url, method, body });
+    const { url, method, contentType, headers, body } = data.payload.settings;
+    log('sendHttp', { url, method, contentType, headers, body });
+
+    let defaultHeaders = contentType ? {
+          'Content-Type':  contentType
+      } : {};
+    let inputHeaders = {};
+
+    if (headers) {
+        const headersArray = headers.split(/\n/);
+
+        for (let i = 0; i < headersArray.length; i += 1) {
+            if (headersArray[i].includes(':')) {
+                const [headerItem, headerItemValue] = headersArray[i].split(/:(.+)/);
```

Comment:
If the line is something like `'foo:'`, this will evaluate to `[ 'foo:', undefined ]`.